_episodes/07-functions.md

```diff
@@ -26,7 +26,7 @@ results.txt : $(ZIPF_SRC) *.dat
 dats : isles.dat abyss.dat last.dat
 
 %.dat : books/%.txt $(COUNT_SRC)
-        $(COUNT_EXE) $< $*.dat
```

Comment:
At this point, we are summarising the previous episode, so we should still using $*.dat - this part should be left as it was.